terraform/variables.tf

```diff
@@ -12,24 +12,30 @@ variable "remote_state_bucket" {
 }
 
 variable "state_prefix" {}
+variable "workspace_key_prefix" {}
+variable "state_file_name" {}
 
-# These vpcs are configured for development. Preprod and prod are overridden in the vars file.
-variable "vpc_cidr" {
-  default = {
-    eu-west-2 = "10.75.0.0/16" #London
-  }
+# Required for access from existing Concourse instances
+variable "dev_management_cidrs" {
+  type    = list(string)
+  default = ["10.50.17.0/24","10.50.19.0/24","10.50.21.0/24"]
 }
-# These vpcs are configured for development. Preprod and prod are overridden in the vars file.
+
+# VPC ID read from terraform remote state.
 variable "vpc_id" {
-  default = {
-    eu-west-2 = "vpc-074ff55ed5182e144" #London
-  }
+  default     = ""
 }
 
 variable "environment" {}
-variable "internal_cidrs" {}
 variable "deploy_to" {}
 
+variable "docker_registry" {} # This value passed in from CI pipeline
```

Comment:
I'd remove these comments. While 99% of the time we'll be running in a pipeline this comment isn't strictly true

Reply:
Removed the "This value passed in from CI pipeline" comments.